src/main/java/frc/robot/commands/Presets/PickupPresetSequence.java

```diff
@@ -28,15 +26,15 @@ public PickupPresetSequence(TelescopeSubsystem telescope, RotatorSubsystem rotat
         else {
             addCommands(crocodile.setWristToPositionCommand(WristPosition.CUBE_PICKUP).withTimeout(2));
         }
-        addCommands(new AutoIntakeCommand(crocodile, 1, button));
+        addCommands(new AutoIntakeCommand(crocodile, 1, button, gamePieice));
     }
-    public PickupPresetSequence(TelescopeSubsystem telescope, RotatorSubsystem rotator, CrocodileSubsystem crocodile, BooleanSupplier button) {
+    public PickupPresetSequence(TelescopeSubsystem telescope, RotatorSubsystem rotator, CrocodileSubsystem crocodile, BooleanSupplier button, boolean gamePieice) {
         addRequirements(telescope,rotator,crocodile);
         addCommands(
             new TelescopeToPosition(telescope, TelescopePosition.GROUND_PICKUP).withTimeout(2),
             new RotatorToPosition(rotator, telescope, RotatorPosition.GROUND_PICKUP).withTimeout(2)
         );
         addCommands(crocodile.setWristToPositionCommand(WristPosition.CONE_PICKUP).withTimeout(2));
-        addCommands(new AutoIntakeCommand(crocodile, 1, button));
+        addCommands(new AutoIntakeCommand(crocodile, 1, gamePieice));
```

Comment:
In `SubstationPickupPreset` you pass in `crocodile.gamepiece` into AutoIntake differently, take a look at that so they are both the same

Reply:
I think I just forgot to change this one

Reply:
i forgot to change the other one